src/pages/Transaction.vue

```diff
@@ -131,7 +131,7 @@ export default {
                                     console.error(`Could not retreive metadata for ${contract.address}: ${e.message}`);
```

Comment:
probably should be sent to the UI and translated as well

Reply:
fixed